src/Stryker.Core/Stryker.Core/Initialisation/ProjectOrchestrator.cs

```diff
@@ -25,20 +26,23 @@ public class ProjectOrchestrator : IProjectOrchestrator
         private readonly ILogger _logger;
         private readonly IBuildalyzerProvider _buildalyzerProvider;
         private readonly IProjectMutator _projectMutator;
+        private readonly IInitialBuildProcess _initialBuildProcess;
 
         public ProjectOrchestrator(IBuildalyzerProvider buildalyzerProvider = null,
-            IProjectMutator projectMutator = null)
+            IProjectMutator projectMutator = null,
+            IInitialBuildProcess initialBuildProcess = null)
         {
             _buildalyzerProvider = buildalyzerProvider ?? new BuildalyzerProvider();
             _projectMutator = projectMutator ?? new ProjectMutator();
+            _initialBuildProcess = initialBuildProcess ?? new InitialBuildProcess();
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<ProjectOrchestrator>();
         }
 
         public IEnumerable<IMutationTestProcess> MutateProjects(StrykerOptions options, IReporter reporters)
         {
-            if (IsSolutionContext(options))
+            if (options.IsSolutionContext)
             {
-                // Build and analyze all projects in the solution with buildalyzer
+                // Analyze all projects in the solution with buildalyzer
                 var solutionProjects = AnalyzeSolution(options);
```

Comment:
```suggestion
                var solutionAnalyzerResults = AnalyzeSolution(options);
```

We need a better name for this, all throughout. solutionProjects is confusing, are they ProjectComponents? ProjectInfo?